compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -605,9 +605,21 @@ struct QuantizeSpecialActivation final : public luci::CircleNodeMutableVisitor<v
   loco::DataType input_type;
   loco::DataType output_type;
 
-  void visit(luci::CircleNode *)
+  void visit(luci::CircleNode *node)
   {
-    // Do nothing by default
+    // Nodes fused with activation functions which need special quantization
+    auto fused_act_node = dynamic_cast<CircleNodeMixin<CircleNodeTrait::FusedActFunc> *>(node);
+    if (fused_act_node != nullptr &&
+        fused_act_node->fusedActivationFunction() == FusedActFunc::TANH)
+    {
+      if (output_type == loco::DataType::U8)
```

Comment:
optional) This code is duplicated with L638. I guess It could be extracted as a separate function.

Reply:
You're right. Let's handle it in another PR.